Re-pull PR #469 (Fix Traditional Chinese (Taiwan) Translation)

jrthsr700tmax opened the pull request:
**Re-pull  PR #469**

**Fix the wrong term in Traditional Chinese (Taiwan) Translation below:
修正正體中文（臺灣）的錯誤翻譯如下：**

- "**收費**" means to **charge money**.
"**充電**" means to **charge battery**.
「收費」在英文意指「charge money」。
「充電」在英文意指「charge battery」。
- Fix the translation `"cycle"` to fit with macOS system's.
修正「`cycle`」的正體中文（臺灣）翻譯來與系統相符。

————————————————————————

You can view, comment on, or merge this pull request online at:
您可以在以下連結檢視、留言或線上合併此Pull Request：

- https://github.com/exelban/stats/pull/481

**Commit Summary
摘要**

- update and optimize Traditional Chinese (Taiwan) translation "last charge"
更新並最佳化正體中文（臺灣）的「last charge」翻譯
- Fix the translation `"cycle"` to fit with macOS system's.
修正「`cycle`」的正體中文（臺灣）翻譯來與系統相符。

**File Changes
檔案變更**

- **M** [Stats/Supporting Files/zh-Hant.lproj/Localizable.strings](https://github.com/exelban/stats/pull/481/files#diff-3e369a5ec4b444abd93a902a19f70e5eb31eb48566ac207fc8e536ea311ae1ca) (4)

**Patch Links:
補丁連結：**

- https://github.com/exelban/stats/pull/481.patch
- https://github.com/exelban/stats/pull/481.diff

exelban commented:
Hi.
Could you please explain what is going on there?
Or have open a 2 PR (this and #469). And it looks like the same, but in #469 you delete the Chinese Traditional translation.

jrthsr700tmax commented:
I'm sorry about that

In cause of the the changing checked failure when I updated the file of PR #469 and I didn't found any way how to fix the problem of check failure. It's the reason that I deleted the file of PR #469 and create this PR.

————————————————————————
IN CHINESE

我很抱歉

由於當時我更新了 PR #469 的檔案，而該更新在檢查時出現錯誤，後續我找不出如何修復該錯誤。因此我刪除了 PR #469 的檔案並重新建立此 PR。

exelban commented:
I don't really understand. You did not find the way how to revert changes in git?
Why you just do not close the previous issue?
